liquibase-couchbase/src/test/java/liquibase/ext/couchbase/change/MutateInChangeTest.java

```diff
@@ -62,6 +64,23 @@ void Should_has_correct_confirm_msg() {
                 .isEqualTo("MutateIn %s operations has been successfully executed", 1);
     }
 
+    @Test
+    void Should_generate_statement_correctly() {
+        MutateInChange change = change(Lists.newArrayList(
+                new LiquibaseMutateInSpec("test", Lists.newArrayList(new Value("data", DataType.STRING)), MutateInType.INSERT)));
```

Comment:
`singletonList()` could be use instead of `newArrayList()`

Reply:
should be done